### DISCUSSION TITLE
Should Eddie go back to light mode for streaming?

### DISCUSSION THREAD

**Discussion by stemount** [EddieHubCommunity/support#630] [General] [answered]
I know from years of accessibility research in government that there's actually an argument for both "dark background, light text", and "light text on a dark background" as it's pretty much split 50:50 on what people find harder to read, so I am open to this being a huge fat NO.

@eddiejaoude However my suggestion is only really about the legibility of text when YouTube realises your internet is struggling a bit to stream, it will lower the bitrate and I find that this turns dark mode websites and code editors into mush.

**This is an example to show the comparison at 720p side by side (scaled)**

![image](https://user-images.githubusercontent.com/150512/103537177-d5d72700-4e8b-11eb-9ae8-a335483cf4d1.png)

**Example of 1080p GitHub**

![image](https://user-images.githubusercontent.com/150512/103537590-93621a00-4e8c-11eb-8088-16aef748a107.png)

![image](https://user-images.githubusercontent.com/150512/103537640-aaa10780-4e8c-11eb-9890-416ba127d013.png)

At the same time though, if you're using 1080p YouTube and the bitrate is quite good, sometimes it corrects itself:

![image](https://user-images.githubusercontent.com/150512/103538773-c5747b80-4e8e-11eb-8a3c-29480edb40fb.png)

I think it should be clear now which horse I back ;)

**Comment by mikeysan** [comment]
Interesting. I have seen what you are referring to in terms of when the bitrate drops. However, for my experience, I would still prefer dark mode.

Here's why. Everything I am on these days from phone to laptop to kindle is all displayed in dark mode. The moment I visit any site that is not in this mode it takes a considerable time for my eyes to adjust.

Actually, maybe that's just age catching up to me. Who knows 🤷

**Reply by Vyvy-vi** [reply]
The same happens with me... 
(or maybe I'm aging faster 🤔  xD)

**Reply by stemount** [reply]
yea I hear you @mikeysan @Vyvy-vi I was wondering what would happen. I've done some further research and my best guess to keep dark mode is to beat the compression™

e.g. a solution for this whole thing is basically whacking up the font size so more pixels change.

**Comment by eddiejaoude** [comment, marked as answer]
Thanks @stemount !! Amazing and super useful feedback ❤️ 

I am happy to switch to light mode for streaming, as well as increase the font size even further. I don't mind light mode for GitHub 🤓

**Reply by stemount** [reply]
Thanks all for the feedback. I think whacking up the font size is the fix overall and I totally hear @mikeysan and @Vyvy-vi with their feedback - and @srinath1412001 with thumbs up!
